.python-version

```diff
@@ -0,0 +1,5 @@
+2.7.14
+3.6.3
+3.3.6
+3.4.7
+3.5.2
```

Comment:
this file was accidentally added because I was working with `pyenv` which proved to be very useful for testing locally with multiple python versions. 

I was going to remove it, but then thought it would be a good to have such convention as repository users, to use pyenv. your call.

Reply:
@alecthomas What do you think about adding this to the package?

Reply:
Let's remove this for now and create an issue where we will continue this discussion.